src/controlnet_aux/dwpose/util.py

```diff
@@ -0,0 +1,334 @@
+import math
```

Comment:
Can you prune this file removing what isn't needed and use versions that already exist in controlnet_aux?

Reply:
Fixed.